app/routes/rental_routes.py

```diff
@@ -0,0 +1,75 @@
+from flask import Blueprint, jsonify, make_response, request, abort
+from app.models.video import Video
+from app.models.customer import Customer
+from app.models.rental import Rental
+from app.routes.customer_routes import get_customer_from_id
+from app.routes.video_routes import is_parameter_found
+from app import db
+from datetime import datetime, timezone, timedelta, date
+import requests
+import os
+
+rental_bp = Blueprint("rentals", __name__, url_prefix="/rentals")
+
+
+@rental_bp.route("/check-out", methods=["POST"])
+def checkout_rental():
+    request_body = request.get_json()
+
+    if "customer_id" not in request_body or "video_id" not in request_body:
+        abort(400, "Request body must include a customer_id and video_id")
+
+    customer = get_customer_from_id(request_body["customer_id"])
+    video = is_parameter_found(Video, request_body["video_id"])
+
+    # if total inventory is empty (unsure if it's testing against 0 or None, couldn't tell on test)
```

Comment:
For instance:

new_rental = Rental.check_out(video, customer)

`check_out` is a class method that returns False if there's no available inventory or returns a `new_rental` object if there is. Lines 26-33 Would be in this class method.